app/assets/javascripts/discourse/app/components/private-message-glyph.gjs

```diff
@@ -0,0 +1,20 @@
+import icon from "discourse-common/helpers/d-icon";
+import i18n from "discourse-common/helpers/i18n";
+
+const PrivateMessageGlyph = <template>
+  {{#if @shouldShow}}
+    {{#if @href}}
+      <a href={{@href}} title={{i18n @title}} aria-label={{i18n @ariaLabel}}>
+        <span class="private-message-glyph-wrapper">
+          {{icon "envelope" class="private-message-glyph"}}
+        </span>
+      </a>
+    {{~else}}
```

Comment:
also, lmao at this single whitespace control character in the sea of whitespace 😄